src/layouts/Sites.tsx

```diff
@@ -140,10 +140,10 @@ const SitesContent = ({ siteNames }: { siteNames?: SiteData[] }) => {
 }
 
 export const Sites = (): JSX.Element => {
-  const { email, userId, contactNumber } = useLoginContext()
+  const { email, contactNumber } = useLoginContext()
   const { data: siteRequestData } = useGetAllSites(email)
-  const { announcements, link } = useAnnouncements()
-  const [isOpen, setIsOpen] = useState(announcements.length > 0 && !userId)
+  const { announcements, link, onCloseButtonText } = useAnnouncements()
+  const [isOpen, setIsOpen] = useState(announcements.length > 0)
```

Comment:
can i ask why we changed the conditionals here?

Reply:
Sure, previously we gated this to only the newer identity folks. 
At the first render of the `Sites.tsx`, we get `userId = undefined` for email users +  `userId = <userId>` for those who are in gh login. 
This announcement does not follow that semantics, and as such this conditional is removed.